docs/map.xsd

```diff
@@ -204,41 +358,92 @@
 <xs:attributeGroup name="property">
   <xs:attribute name="name" type="nameT" use="required"/>
   <xs:attribute name="value" type="xs:string" use="required"/>
+  <xs:attribute name="type" type="propertytypeT" use="optional" default="string"/>
 </xs:attributeGroup>
 
 <xs:attributeGroup name="map">
   <xs:attribute name="version" type="xs:string" use="required"/>
   <xs:attribute name="orientation" type="orientationT" use="required"/>
-  <xs:attribute name="renderorder" type="renderorderT"/>
+  <xs:attribute name="renderorder" type="renderorderT" default="right-down"/>
   <xs:attribute name="width" type="xs:nonNegativeInteger" use="required"/>
   <xs:attribute name="height" type="xs:nonNegativeInteger" use="required"/>
-  <xs:attribute name="tilewidth" type="xs:nonNegativeInteger" use="required"/>
-  <xs:attribute name="tileheight" type="xs:nonNegativeInteger" use="required"/>
+  <xs:attribute name="tilewidth" type="pixelT" use="required"/>
+  <xs:attribute name="tileheight" type="pixelT" use="required"/>
+  <xs:attribute name="hexsidelength" type="pixelT" use="optional"/>
+  <xs:attribute name="staggeraxis" type="staggeraxisT" use="optional"/>
+  <xs:attribute name="staggerindex" type="staggerindexT" use="optional"/>
+  <xs:attribute name="backgroundcolor" type="hexColourAlphaT" use="optional"/>
```

Comment:
Actually, in any place where a `hexColourAlphaT` may be used, a `hexColourT` would also be valid (the alpha is then assumed to be 255). Could that be represented? Maybe as `#([a-fA-F0-9][a-fA-F0-9]){3-4}`

Also, maybe we should rename these to use US English `Color`?